stix_shifter/utils/error_response.py

```diff
@@ -7,6 +7,12 @@
 import json
 
 class ErrorCode(Enum):
+    TRANSLATION_NOTIMPLEMENTED_MODE = 100
+    TRANSLATION_MODULE_DEFAULT_ERROR =  200
+    TRANSLATION_MAPPING_ERROR = 310
+    TRANSLATION_STIX_VALIDATION = 320
```

Comment:
How were these error codes decided? Wouldn't a pattern that fails validation be in the 400 range since it's a poorly formatted request?

Reply:
these codes are not http codes, mapping to http codes will be done in other places.